jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -119,9 +118,8 @@ async def put_filing_on_queue(filing_id: int, app: Flask, qsm: QueueService):
         await qsm.publish_json_to_subject(payload, subject)
     except Exception as err:  # pylint: disable=broad-except # noqa F841;
         # mark any failure for human review
-        capture_message(
-            f'Queue Error: Failed to place filing {filing_id} on Queue with error:{err}',
-            level='error'
+        app.logger.error(
+            f'Queue Error: Failed to place filing {filing_id} on Queue with error:{err}'
```

Comment:
Not 100% what `capture_message` does, but I wasn't seeing any output to the terminal, causing `put_filing_on_queue` to fail silently. The reason it was failing is that the NATS_FILER_SUBJECT was not set in the config.

Reply:
The `capture_message` is for sentry.

Lets leave the `capture_message` and add the `app.logger.error` statement as well.

Reply:
Added back the capture_message